docs/usage/security-and-permissions.md

```diff
@@ -1,6 +1,33 @@
 # Security and Permissions
 
-## Global Permissions
+This page talks about our security stance, and explains what permissions are needed for the different ways you can run Renovate.
+
+## Security Stance
+
+Renovate is open source software, and comes with no guarantees or warranties of any kind.
+That said, we will try to fix security problems in a reasonable timeframe if possible.
+
+If you need commercial level support then use INSERT OUR COMMERCIAL OFFERING HERE.
+
+### No certifications
+
+Renovate does **not** have ISO 27001 or SOC2 certification.
+
+### How to report a security problem
+
+If you discover any important bug with Renovate that may pose a security problem, please disclose it confidentially to [renovate-disclosure@whitesourcesoftware.com](mailto:renovate-disclosure@whitesourcesoftware.com) first, so that it can be assessed and hopefully fixed prior to being exploited.
+Please do not raise GitHub issues for security-related doubts or problems.
+
+## Permissions
+
+We only request those permissions we need for our apps to do useful work.
+If we do not need a specific permission, we will not request it.
+
+We also limit each permission to the lowest level possible.
```

Comment:
Please check if I'm describing our stance on permissions correctly here.

Reply:
I think you could summarize it as "We apply the Principle of Least Authority (POLA) but do need substantial privileges in order to perform our functionality"

Reply:
```suggestion
We apply the Principle of Least Privilege (PoLP) but do need substantial privileges in order for our apps to work.
```

How about something like this?

I don't know which of these is most commonly used:

- Principle of Least Authority
- Principle of Least Privilege
- Principle of Minimal Privilege